geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisServer.java

```diff
@@ -79,9 +80,12 @@ public GeodeRedisServer(String bindAddress, int port, InternalCache cache) {
     redisStats = createStats(cache);
     StripedExecutor stripedExecutor = new SynchronizedStripedExecutor();
     regionProvider = new RegionProvider(cache);
+    CommandHelper commandHelper =
+        new CommandHelper(regionProvider.getDataRegion(), redisStats, stripedExecutor);
 
     CommandFunction.register(regionProvider.getDataRegion(), stripedExecutor, redisStats);
     RenameFunction.register(regionProvider.getDataRegion(), stripedExecutor, redisStats);
+    RedisMemberInfoRetrievalFunction infoFunction = RedisMemberInfoRetrievalFunction.register();
```

Comment:
I think we need to make sure this is registered before the data region is created, so probably before the regionProvider initialization?